packages/http-client/src/http-client-types.ts

```diff
@@ -88,10 +88,16 @@ interface KnownHttpResponseFields {
   request: any;
 }
 
+/**
+ * This interface is compatible with AxiosResponse.
+ */
```

Comment:
[nth] This is nice information for a second sentence, but not the main description. I miss something that describes what this is.